server_admin/topics/user-federation/ldap.adoc

```diff
@@ -137,6 +137,14 @@ FullName Mapper::
   This allows you to specify that the full name of the user, which is saved in some LDAP attribute (usually `cn` ) will be mapped to `firstName` and `lastname` attributes in the {project_name} database.
   Having `cn` to contain full name of user is a common case for some LDAP deployments.
 
+NOTE: When registering new users in {project_name} and `Sync Registrations` switch is ON for the LDAP provider, the fullName mapper
+      allows the possibility of fallback to the username. This is especially useful in case of the Microsoft Active Directory. The common
```

Comment:
Try to always follow "this" with a noun, so that "this" is clear what it refers to.

So "this fallback..."

Reply:
Good point. I will try to improve on this and hopefully next time, I will finally do it right :)

Reply:
Okay, sounds good. Thanks, @mposolda